### ISSUE TITLE
Music Library Scan Issue

### ISSUE THREAD

**Issue by igbjnI05bF** [jellyfin/jellyfin#63]
On a fresh Debian9 VM, after having installed Jellyfin from the repo, I am unable to get the Music library to scan correctly. Here is a tail -n 50 of server.txt: https://paste.pound-python.org/show/GzsZomKLjz8W2bWvmrJv/ (that repeats over and over during scan)

Songs show up but nothing in Artists at all.

**Comment by BnMcG** [comment]
Errors in the log excerpt provided come from this method: https://github.com/jellyfin/jellyfin/blob/21a8d273782a1a71678131f17ebaafd2b997f286/Emby.Server.Implementations/Updates/InstallationManager.cs#L214 

Looks like it's failing to phone home (expected). Can you paste the whole log? I don't think this should stop library scanning working.

**Comment by igbjnI05bF** [comment]
Here is a "head -n 100000" of server.txt on a fresh install, adding only my music folder. Anything more than this is too large to upload
[server1.txt](https://github.com/jellyfin/jellyfin/files/2669923/server1.txt)

**Comment by BnMcG** [comment]
OK, this looks like it could be the problem (or at least, partially):

```
2018-12-11 17:56:52.630 Info MediaEncoder: Running ffmpeg -version
2018-12-11 17:56:52.649 Error MediaEncoder: Error validating encoder
	*** Error Report ***
	Version: 3.5.2.0
	Command line: /usr/lib/jellyfin/bin/EmbyServer.dll
	Operating system: Unix 4.9.0.8
	64-Bit OS: True
	64-Bit Process: True
	User Interactive: True
	Processor count: 2
	Program data path: /var/lib/emby/
	Application directory: /usr/lib/jellyfin/bin
	System.ComponentModel.Win32Exception (2): No such file or directory
	   at System.Diagnostics.Process.StartCore(ProcessStartInfo startInfo)
	   at System.Diagnostics.Process.Start()
	   at Emby.Server.Implementations.Diagnostics.CommonProcess.Start() in /var/home/joshua/Projects/Emby/jellyfin/Emby.Server.Implementations/Diagnostics/CommonProcess.cs:line 112
	   at Emby.Server.MediaEncoding.Encoder.EncoderValidator.GetProcessOutput(String path, String arguments)
	   at Emby.Server.MediaEncoding.Encoder.EncoderValidator.ValidateVersion(String encoderAppPath, Boolean logOutput)
	System.ComponentModel.Win32Exception
	   at System.Diagnostics.Process.StartCore(ProcessStartInfo startInfo)
	   at System.Diagnostics.Process.Start()
	   at Emby.Server.Implementations.Diagnostics.CommonProcess.Start() in /var/home/joshua/Projects/Emby/jellyfin/Emby.Server.Implementations/Diagnostics/CommonProcess.cs:line 112
	   at Emby.Server.MediaEncoding.Encoder.EncoderValidator.GetProcessOutput(String path, String arguments)
	   at Emby.Server.MediaEncoding.Encoder.EncoderValidator.ValidateVersion(String encoderAppPath, Boolean logOutput)
	
2018-12-11 17:56:52.649 Info MediaEncoder: FFMpeg: not found
2018-12-11 17:56:52.649 Info MediaEncoder: FFProbe: not found
2018-12-11 17:56:52.652 Info App: ServerId: b39e69845b3c45b68aa6d2a79ab181b4
2018-12-11 17:56:52.689 Info App: Starting entry point Emby.Dlna.Main.DlnaEntryPoint
2018-12-11 17:56:52.832 Info App: Entry point completed: Emby.Dlna.Main.DlnaEntryPoint. Duration: 0.1434292 seconds
```

**Comment by BnMcG** [comment]
Following up from Riot chat, it seems like ffmpeg may not have been installed.

**Comment by igbjnI05bF** [comment]
Installing ffmpeg does appear to have resolved the issue. Everything is still scanning but Artists are now popping up. Thanks for your assistance, BnMcG. Perhaps include ffmpeg as a dependency for jellyfin in debian? I'll mark this as closed.

**Comment by JustAMan** [comment]
Should be fixed by #71
